app/workflows/hearing_request_case_distributor.rb

```diff
@@ -13,13 +13,18 @@ def initialize(appeals:, genpop:, distribution:, priority:)
   def call
     appeals_to_distribute.map do |appeal, genpop_value|
       Distribution.transaction do
+        redistribute_existing_distributed_case(appeal)
```

Comment:
```suggestion
        rename_any_existing_distributed_case(appeal)
```
We should rename the `DistributedCase#redistribute` method -- it doesn't actually redistribute.